docs/_pages/releases.md

```diff
@@ -9,6 +9,8 @@ sidebar:
 
 ## 8.4.0
 
+* Added `Should().Throw()`, `ThrowAsync()` and `ThrowWithinAsync()` flavors that don’t require a specific exception type - [3059](https://github.com/fluentassertions/fluentassertions/pull/3059)
```

Comment:
🔧 This should be placed under the "enhancements" section. See also previous versions.